mlpp_lib/probabilistic_layers.py

```diff
@@ -153,18 +263,18 @@ def get_config(self):
             "convert_to_tensor_fn": _serialize(self._convert_to_tensor_fn),
             "validate_args": self._validate_args,
         }
-        base_config = super(IndependentBeta, self).get_config()
+        base_config = super(Independent4ParamsBeta, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
 
     @property
     def output(self):
         """This allows the use of this layer with the shap package."""
-        return super(IndependentBeta, self).output[0]
+        return super(Independent4ParamsBeta, self).output[0]
 
 
 @tf.keras.saving.register_keras_serializable()
-class IndependentGamma(tfpl.DistributionLambda):
-    """An independent gamma Keras layer."""
+class IndependentCensoredNormal(tfpl.DistributionLambda):
```

Comment:
keep only one censored normal, either this one or the one based on the clipping bijector